test/PHPMailerTest.php

```diff
@@ -2675,6 +2675,131 @@ public function testOAuth()
         $this->assertNull($subject);
         $this->assertInstanceOf(OAuth::class, $PHPMailer->getOAuth());
     }
+
+    /**
+     * Test ICal method
+     */
+    public function testICalMethod()
+    {
+        $this->Mail->Subject .= ': ICal method';
+        $this->Mail->Body = '<h3>ICal method test.</h3>';
+        $this->Mail->AltBody = 'ICal method test.';
+        $this->Mail->Ical = 'BEGIN:VCALENDAR'
+            . '\r\nVERSION:2.0'
```

Comment:
These `\r\n` sequences won't work because you're using single quotes, and those chars won't get unescaped; changing to double quotes will fix that.